### ISSUE TITLE
Add `no-unused-imports`?

### ISSUE THREAD

**Issue by ebebbington** [denoland/deno_lint#598]
Are there plans to add this rule? I know personally it'd be amazing if `deno lint` could also throw errors/warnings on unused imports

**Comment by magurotuna** [comment]
This should be covered by `no-unused-vars`. In fact, if this particular rule is enabled, the following snippet will give us a lint error.

```ts
import { foo } from "./aaa.ts";

console.log(42);
```

output:

```
error[no-unused-vars]: "foo" is never used
 --> foo.ts:1:10
  |
1 | import { foo } from "./aaa.ts";
  |          ^^^
  |
error[no-unused-vars]: "foo" is never used
 --> foo.ts:1:10
  |
1 | import { foo } from "./aaa.ts";
  |          ^^^
  |
Found 2 problems
```

(It might be a bug to get duplicate errors, but ignore it for now)
I don't have an idea why `no-unused-vars` is not listed in the "recommended" set. I guess the reason is that the rule is prone to have false positives.
@bartlomieju do you know anything about it?

**Comment by bartlomieju** [comment]
@magurotuna I'm not sure why it's not in recommended set. I guess it would make sense to add it.

**Comment by ebebbington** [comment]
That would be amazing if it’s part of the default set!!! 😊
